backend/app/services/prompt_draft/modeling.py

```
@@ -0,0 +1,65 @@
+    db: Session,
+    current_user: User,
+    requested_model_name: str | None,
+) -> tuple[dict[str, Any] | None, str]:
+    model_kind = None
+    if requested_model_name:
+        model_kind = (
+            db.query(Kind)
+            .filter(
+                Kind.kind == "Model",
+                Kind.name == requested_model_name,
+                Kind.is_active == True,
```

Ruff (0.15.7)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)